index.html

```diff
@@ -112,11 +112,22 @@
 </header>
 
 <div id="main">
-	<p>
-	Help us help you: put in your code along with a brief description about what you are trying to do and how it failed (such as error messages).
-	For installation and project management questions, include stack.yaml, *.cabal, cabal.project files (if you have any of those, of course).
-	If you have 5 minutes to spare, please enjoy <a target="_blank" href="https://www.chiark.greenend.org.uk/~sgtatham/bugs.html">this classic read</a> first; while it is about bug reports, asking good questions is a quite similar process.
-	</p>
+
+<h2 id="help-us-help-you">Help us help you</h2>
+<p>Include in your paste:</p>
+<ul>
+<li>your code</li>
+<li>a brief description for what you are trying to do</li>
+<li>how it failed such as error messages.</li>
+</ul>
+<p>Please enjoy <a href="https://www.chiark.greenend.org.uk/~sgtatham/bugs.html">this</a> classic 5-min read. It is about bug reports, but asking good questions is quite a similar process.</p>
+<h2 id="for-installation-and-project-management">For installation and project management</h2>
+<p>Include separately in your paste (if you have them):</p>
+<ul>
+<li>stack.yaml</li>
+<li>*.cabal,</li>
```

Comment:
It seems Github web interface doesn't allow directly amending or adding another comment on top of a pull request... Line 128 has an extra comma "*.cabal,"
Not sure if we'd keep commas after all the items or not...